Media/Translations/Default.xml

```diff
@@ -148,6 +148,19 @@
 		<ru>Невозможно загрузить карту, переход к следующей.</ru>
 		<sk>Táto mapa nie je platná, prechádzam na ďalšiu.</sk>
 	</segment>
+	<segment>
```

Comment:
A segment with identical meaning already exists. It looks like this:
> This map is invalid; moving to the next one.
Please remove this segment and use the mentioned one in your code.